tests/test-solr.php

```diff
@@ -170,6 +170,32 @@ public function test_index_site_for_multisite() {
         $this->assertEquals(0, $indexResult);
     }
 
+    /**
+     * @group multisite
+     */
+    public function test_index_site_indexes_more_than_first_100_sites() {
+        for ($i = 0; $i < 110; $i++) {
+            $blogId = $this->factory->blog->create();
+
+            switch_to_blog($blogId);
+            $this->factory->post->create_many(1);
+        }
+
+        $solrAdmin = new Ed_Solr_Admin('ed-solr', '1.0.0');
```

Comment:
Wondering if this test is actually doing what we think it is. I think when we create posts, they are automatically indexed because of other features in this plugin. Running the index_all_blogs_in_solr shouldn't make a difference as they are already all indexed. Is this correct? Sorry if I have made a mistake. If you could double check this that would be great thanks.

Reply:
We are just checking that get_sites (called by index_all_blogs_in_solr) isn't being limited to the first 100 blogs. If it was we would only get 200 posts back. Previously, index all blogs only indexed the first 100 blogs. It should now index the first 200000.

Reply:
I think we need to raise an issue for this test in the interest of time. We need to disable the post hook that is indexing individual blogs prior to creating posts to ensure the `index_all_blogs_in_solr()` function is doing its job. Cheers Richard.